src-tauri/build.rs

```diff
@@ -1,3 +1,8 @@
 fn main() {
```

Comment:
Panics early before any building actually begins...

Reply:
This looks good!